frontend_new/src/App.tsx

```diff
@@ -8,8 +8,9 @@ import Search from "./components/Search";
 import Grid from "@material-ui/core/Grid";
 import Typography from "@material-ui/core/Typography";
 import FrontPageInfo from "./components/FrontPageInfo";
-
 import Grow from "@material-ui/core/Grow";
+import MediaCard from "./components/Logo";
```

Comment:
**[Required]** Unused import, can be removed.

On a broader note, there probably don't need to be any changes to this file.